modules/rostests/apitests/atl/CImage_WIP.txt

```diff
@@ -1,86 +0,0 @@
->atl_apitest.exe CImage
-CImage.cpp:234: Test failed: Expected bpp to be 8, was: 32
-CImage.cpp:265: Test failed: Expected bpp to be 8, was: 32 (for 0)
-CImage.cpp:245: Test failed: Expected hr to be S_OK, was: 80004005 (for 1)
-CImage.cpp:251: Test failed: Expected hr to be S_OK, was: 80004005 (for 1)
-CImage.cpp:254: Test failed: Expected width to be 48, was: 0 (for 1)
-CImage.cpp:256: Test failed: Expected height to be 48, was: 0 (for 1)
-CImage.cpp:265: Test failed: Expected bpp to be 8, was: 0 (for 1)
-CImage.cpp:148: Test failed: Expected status to be 0, was: 1
-CImage.cpp:149: Test failed: Expected a valid bitmap
-CImage.cpp:245: Test failed: Expected hr to be S_OK, was: 80004005 (for 2)
-CImage.cpp:251: Test failed: Expected hr to be S_OK, was: 80004005 (for 2)
-CImage.cpp:254: Test failed: Expected width to be 48, was: 0 (for 2)
-CImage.cpp:256: Test failed: Expected height to be 48, was: 0 (for 2)
-CImage.cpp:265: Test failed: Expected bpp to be 8, was: 0 (for 2)
-CImage.cpp:148: Test failed: Expected status to be 0, was: 1
-CImage.cpp:149: Test failed: Expected a valid bitmap
-CImage.cpp:260: Test failed: Expected bpp to be 24, was: 32 (for 3)
-CImage.cpp:154: Test failed: Expected PixelFormat to be 0x21808, was: 0x30803
-CImage.cpp:265: Test failed: Expected bpp to be 8, was: 32 (for 4)
-CImage.cpp:154: Test failed: Expected PixelFormat to be 0x30803, was: 0x21808
-
-CImage: 77 tests executed (0 marked as todo, 20 failures), 0 skipped.
-================================================================================================================
->CImage.exe
-../CImage.cpp (245): Expected hr to be S_OK, was: 80004005 (for 1)
-../CImage.cpp (251): Expected hr to be S_OK, was: 80004005 (for 1)
-../CImage.cpp (254): Expected width to be 48, was: 0 (for 1)
-../CImage.cpp (256): Expected height to be 48, was: 0 (for 1)
-../CImage.cpp (265): Expected bpp to be 8, was: 0 (for 1)
-../CImage.cpp (148): Expected status to be 0, was: 1
-../CImage.cpp (149): Expected a valid bitmap
-../CImage.cpp (245): Expected hr to be S_OK, was: 80004005 (for 2)
-../CImage.cpp (251): Expected hr to be S_OK, was: 80004005 (for 2)
-../CImage.cpp (254): Expected width to be 48, was: 0 (for 2)
-../CImage.cpp (256): Expected height to be 48, was: 0 (for 2)
-../CImage.cpp (265): Expected bpp to be 8, was: 0 (for 2)
-../CImage.cpp (148): Expected status to be 0, was: 1
-../CImage.cpp (149): Expected a valid bitmap
-../CImage.cpp (260): Expected bpp to be 24, was: 8 (for 3)
-../CImage.cpp (154): Expected PixelFormat to be 0x21808, was: 0x30803
-../CImage.cpp (265): Expected bpp to be 8, was: 24 (for 4)
-../CImage.cpp (154): Expected PixelFormat to be 0x30803, was: 0x21808
-CImage: 77 tests executed (0 marked as todo, 18 failures), 0 skipped.
-================================================================================================================
-
```

Comment:
So if I understand correctly, now all those failures have been resolved?

Reply:
`CImage` tests are fixed. It's safe to delete.